### PR TITLE
Sync sparrow task status with celery

### PR DESCRIPTION
**Pull request by wladekb**
This change will automatically update sparrow tasks status when celery starts or finishes processing test.

/cc @harnash @jcellary

### CODE REVIEW

File: apps/tasks/signals.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+task_status_changed = django.dispatch.Signal(providing_args=['instance', ])
+celery_request_status_update = django.dispatch.Signal(providing_args=['task_id', 'job_id', 'status'])
+
+
+@receiver(celery_request_status_update)
+def on_task_manager_request_status_update(sender, task_id, job_id, status, **kwargs):
+    logger.debug('on_task_manager_request_status_update {} {}'.format(job_id, status))
+
+    from tasks.models import Task, TaskStatus
```
Comment (harnash) [inline apps/tasks/signals.py]: Shouldn't we use REST API for this? Theoretically task runner can be on separate machine.

**Comment by harnash** [comment]
Would be nice to have some tests for that.

**Comment by wladekb** [comment]
Any specific idea for a test scenario? Or idea for enhancing existing ones?

**Comment by harnash** [comment]
Well. I would simulate positive and negative paths by just manipulating with uri's and perhaps configuration.  We could create just tasks (in tests only) which will just go through all the possible states.

**Pull request merged**